web/src/use/usePaginatedResults.ts

```diff
@@ -43,11 +43,6 @@ export default function usePaginatedResult<T>(
 
   const debouncedFetchResults = debounce(fetchResults, 500);
 
-  watch([
-    toRef(data, 'limit'),
-    toRef(data, 'offset'),
-  ], debouncedFetchResults);
-
   watch([conditions], () => {
```

Comment:
I removed this because it seemed to make more sense to call directly when updated below

Reply:
Makes sense to me. This appears to be the only place where these are set.